### PR TITLE
Phase2-hgx312E Change names of variabes to enable easy reading of the code for creation of HGCal geometry

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Change names of variabes to enable easy reading of the code for creation of HGCal geometry

#### PR validation:

Use the runTheMatrix test workflows 

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
Thanks for the clarifications.

**Comment by cvuosalo** [comment]
+1

File: Geometry/HGCalCommonData/plugins/DDHGCalMixRotatedLayer.cc [hunk 1]
```diff
@@ -95,7 +95,7 @@ class DDHGCalMixRotatedLayer : public DDAlgorithm {
   std::vector<double> layerThickTop_;     // Thickness of the top sections
   std::vector<int> layerTypeTop_;         // Type of the Top layer
   std::vector<int> copyNumberTop_;        // Initial copy numbers (top section)
-  std::vector<int> layerTypes_;           // Layer type of silicon layers
+  std::vector<int> layerOrient_;          // Layer type of silicon layers
```
[the remark this answers is not in the document]
Comment (bsunanda) [reply Geometry/HGCalCommonData/plugins/DDHGCalMixRotatedLayer.cc]: done

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by srimanob** [comment]
+Upgrade

Technical PR. No change is expected, and not seen in PR test. The failure in comparison is not related to this PR.

**Comment by bsunanda** [comment]
@cvuosalo Could you please sign this - just changed the documentation as suggested by Phat

**Comment by civanch** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**